MailService Integration

Jordex opened the pull request.

Jordex committed:
MailService Integration

Used if statement to differentiate between employee and manager mail

maxim-tschumak commented on backend/src/main/java/de/tuberlin/enterprisecomputing/integrations/MailService.java:
```diff
@@ -1,18 +1,44 @@
     public MailService(){
-        //TODO initialize SES client
+        //TODO initialize SES client, set region
+    	Region REGION = Region.getRegion(Regions.US_WEST_2);  
```
I think we have our Beanstalk in EU_WEST_1 (Ireland)

maxim-tschumak commented:
Doesn't compile. Please check the dependencies in pom.xml. I think you forgot to include the AWS SDKs  dependencies. (for SES, S3 and DynamoDB)

Jordex commented:
I added the dependencies (for SES, S3 and DynamoDB) manually. but when doing 'mvn clean package' no .jar file is created in the target file. Any idea what else could be wrong?

maxim-tschumak commented:
You have to add the dependencies in the project's pom.xml. Maven will download them and add them to classpath then.

---- JonathanHeiss wrote ----

> I added the dependencies (for SES, S3 and DynamoDB) manually. but when doing 'mvn clean package' no .jar file is created in the target file. Any idea what else could be wrong?
> 
> —
> Reply to this email directly or view it on GitHub.￼

Jordex committed:
next try ;)

@mailService: changed region
@POM: added dependencies
@S3Service: getFile() delivers byte[] instead of Byte[], in storeFile()
type 'file' is passed instead of Byte[]

maxim-tschumak commented:
looks good!
